minutes/2021-09-02 Synchronous GA Meeting.md

```diff
@@ -0,0 +1,45 @@
+# Synchronous Vote GA: Training Sessions
+
+## Attendees:
+
+Jeff Long
+Marc Lichtman
+Barry Duggan
+Martin Braun
+Nate Temple
+Johannes Demel
+Marcus Mueller
+Samantha Palazzolo
+Philip Balister
+Andrej Rode
+Bastian Bloessl
+Derek Kozel
+Josh Morman
+
+
+## Agenda:
+1. Discussion of the Board and GA activities (see report)
+  - This is an open discussion. Feedback is particularly welcome!
+2. Elections of the new Board.
+  - Elections are handled by our MCs, Barry and Josh.
+  - Currently, Derek, Marc, and myself have decided to run for re-election.
```

Comment:
myself > Martin

Reply:
lol - copy and paste from Martin's email

Reply:
hah np